claim-check/code-samples/sample-1/client-consumer/ClientConsumer.csproj

```diff
@@ -10,7 +10,7 @@
   <ItemGroup>
     <PackageReference Include="Azure.Storage.Blobs" Version="12.1.0" />
     <PackageReference Include="Azure.Storage.Queues" Version="12.1.0" />
-    <PackageReference Include="Microsoft.Azure.ServiceBus" Version="3.3.0" />
+    <PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
```

Comment:
Is newtonsoft.json being used in this project?    Prefer System.Text.Json if you need Json capabilities.

Reply:
Yes, it was being used in `StorageQueueConsumer.cs` file.

Reply:
can we use System.Text.Json instead?

Reply:
I have replaced `Newtonsoft.Json` with `System.Text.Json`.